inc/cookie/namespace.php

```diff
@@ -19,8 +19,12 @@ function is_enabled() : bool {
  * Login page footer for the login link.
  */
 function on_login_form() {
+	$login_url = get_authorize_url();
```

Comment:
Hmm so turns out I think I did this bit wrong. The `get_authorize_url` already contains the `redirect_uri` value, which I need to append `redirect_to` to, rather than another get param altogether. So, I change is actually needed to `get_authorize_url`. I had incorrectly verified this to be the wrong URL.